CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
 
 ### Enhancements:
 
+- Add cloud storage backups to AWS S3 for aim repo runs(karan2801)
+
+### Enhancements:
```

Comment:
it seems that the `Enhancements` title is duplicated here. the section should be like this:

### Enhancements:

- Add cloud storage backups to AWS S3 for aim repo runs(karan2801)
- Add descriptive document titles for pages (KaroMourad)
- Implement unit-tests for aim SDK utils (yeghiakoronian)
- ...